Dev/src/screens/AuthenticationScreens/LoginScreen.js

```diff
@@ -65,23 +65,27 @@ class LoginScreen extends Component {
     this.setState({ isLoading: true });
     const { username, password } = this.state;
 
-    const account = await FirebaseFunctions.logIn(username.trim(), password.trim());
-    if (account === -1) {
-      this.setState({ isLoading: false });
-      Alert.alert(strings.Whoops, strings.IncorrectInfo);
+    if (username.trim() === "" || password.trim() === "") {
+      Alert.alert(string.Whoops, strings.PleaseMakeSureAllFieldsAreFilledOut);
     } else {
-      const userID = account.uid;
-      if (this.state.isTeacher === true) {
-        FirebaseFunctions.logEvent("TEACHER_LOG_IN");
-        this.props.navigation.push("TeacherCurrentClass", {
-          userID,
-        })
+      const account = await FirebaseFunctions.logIn(username.trim(), password.trim());
```

Comment:
You should not trim the password. It may correctly contain a space in the beginning or the end.